testsuite/python/mmm1d.py

```diff
@@ -67,11 +67,11 @@ def test_energy(self):
 
     def test_with_analytical_result(self, prefactor=1.0, accuracy=1e-4):
         self.system.part.clear()
-        self.system.part.add(pos=[0, 0, 0], q=1)
-        self.system.part.add(pos=[0, 0, 1], q=1)
+        p = self.system.part.add(pos=[0, 0, 0], q=1)
+        _ = self.system.part.add(pos=[0, 0, 1], q=1)
```

Comment:
Is there a reason to save the handle? aesthetics?
```suggestion
        self.system.part.add(pos=[0, 0, 1], q=1)
```

Reply:
`_` has a special meaning in Python and is often used to discard temporary values, in this case I used if for aesthetics but I wouldn't mind removing it

Reply:
I know about the convention, I only use it for tuple-unpacking. I was pointing it out because there are other occasions with the same situation
https://github.com/espressomd/espresso/blob/2293827912cfc8ab0657aadd11455c179a11eef7/testsuite/python/interactions_bond_angle.py#L41-L43